codechecker_lib/analyzers/analyzer_types.py

```diff
@@ -299,14 +289,11 @@ def build_config_handlers(args, context, enabled_analyzers, connection=None):
     for ea in enabled_analyzers:
         if ea == CLANG_SA:
             config_handler = __build_clangsa_config_handler(args, context)
-            analyzer_config_map[ea] = config_handler
-
         elif ea == CLANG_TIDY:
             config_handler = __build_clang_tidy_config_handler(args, context)
-            analyzer_config_map[ea] = config_handler
         else:
-            LOG.debug_analyzer('Not supported analyzer type. '
-                               'No configuration handler will be created.')
+            assert False, 'Analyzer types should have been checked already.'
```

Comment:
Should be the default value of the config_handler None?
Do we need to fail with an assert if it is misconfigured? The basic idea was that if a non supported analyzer is configured just skip it and run the supported and available analyzers.